Gems/EMotionFX/Code/EMotionFX/Rendering/Common/RenderUtil.cpp

```diff
@@ -238,12 +238,12 @@ namespace MCommon
 
 
     // render selection gizmo around the given AABB
-    void RenderUtil::RenderSelection(const MCore::AABB& box, const MCore::RGBAColor& color, bool directlyRender)
+    void RenderUtil::RenderSelection(const AZ::Aabb& box, const MCore::RGBAColor& color, bool directlyRender)
     {
-        //const Vector3 center  = box.CalcMiddle();
-        const AZ::Vector3       min     = box.GetMin();// + (box.GetMin()-center).Normalized()*0.005f;
-        const AZ::Vector3       max     = box.GetMax();// + (box.GetMax()-center).Normalized()*0.005f;
-        const float             scale   = box.CalcRadius() * 0.1f;
+        const AZ::Vector3       min     = box.GetMin();
+        const AZ::Vector3       max     = box.GetMax();
+        const float             radius  = AZ::Vector3(box.GetMax() - box.GetMin()).GetLength() * 0.5f;
+        const float             scale   = radius * 0.1f;
         const AZ::Vector3       up      = AZ::Vector3(0.0f, 1.0f, 0.0f) * scale;
         const AZ::Vector3       right   = AZ::Vector3(1.0f, 0.0f, 0.0f) * scale;
         const AZ::Vector3       front   = AZ::Vector3(0.0f, 0.0f, 1.0f) * scale;
```

Comment:
Nit: `p` below, on line 252, can be `const`, if it is declared like this:

```
const AZStd::array p {
    AZ::Vector3{min.GetX(), min.GetY(), min.GetZ()},
    AZ::Vector3{max.GetX(), min.GetY(), min.GetZ()},
    AZ::Vector3{max.GetX(), min.GetY(), max.GetZ()},
    AZ::Vector3{min.GetX(), min.GetY(), max.GetZ()},
    AZ::Vector3{min.GetX(), max.GetY(), min.GetZ()},
    AZ::Vector3{max.GetX(), max.GetY(), min.GetZ()},
    AZ::Vector3{max.GetX(), max.GetY(), max.GetZ()},
    AZ::Vector3{min.GetX(), max.GetY(), max.GetZ()},
};
```

Reply:
Changed